frontend/coprs_frontend/coprs/logic/builds_logic.py

```diff
@@ -1034,11 +1034,12 @@ def finish(chroot, status):
                 for chroot in build.build_chroots:
                     arch = chroot.mock_chroot.arch
 
-                    exclusivearch = upd_dict["results"]["exclusivearch"]
+                    exclusivearch = upd_dict["results"].get("exclusivearch")
                     if exclusivearch and arch not in exclusivearch:
                         finish(chroot, StatusEnum("skipped"))
 
-                    if arch in upd_dict["results"]["excludearch"]:
+                    excludearch = upd_dict["results"].get("excludearch")
```

Comment:
The change is okay and will fix the traceback, so +1.
But IMHO the `"exclusivearch"` and `"excludearch"` should always be present, so there is probably a bug earlier in the pipeline. Per @praiskup on Slack, it seems that we are using `RPMResults` instead of `SRPMResults` for custom method.

Reply:
Should I drop the commit?  Or what do you prefer?